src/Manager.js

```diff
@@ -1,27 +1,61 @@
 import User from './User'
 
 class Manager extends User {
-  constructor(date, rooms, bookings) {
-    super(id);
-    this.date = date;
+  constructor(customers, rooms, id, date, bookings) {
```

Comment:
I think you'll need to put rooms at the end of your constructor params for Manager as well. I am curious if all your tests are passing as it is? Maybe I am confused about how supers work?? But I think you want to be careful of order for these when using a parent class.